mage_ai/io/redshift.py

```diff
@@ -227,16 +226,26 @@ def __process():
                     cur.execute(query)
                 else:
                     if should_create_table:
-                        col_with_types = ', '.join(
-                            [f'{col} {col_type}' for col, col_type in columns_with_type],
-                        )
+                        if overwrite_types is not None:
+                            for index, col in enumerate(columns_with_type):
+                                # col is a tuple with (col_name, col_type)
+                                # so we access col_name using col[0]
+                                if col[0] in overwrite_types.keys():
+                                    columns_with_type[index] = (col[0], overwrite_types[col[0]])
+
+                        col_with_types = [f'{col} {col_type}' for col, col_type in
+                                          columns_with_type]
+
+                        col_with_types = ', '.join(col_with_types)
                         query = f'CREATE TABLE IF NOT EXISTS {full_table_name} ({col_with_types})'
                         cur.execute(query)
 
                     columns = ', '.join([t[0] for t in columns_with_type])
                     values = [f"""({', '.join([format_value(x) for x in v])})""" for v in df.values]
                     values = ', '.join(values)
                     query = f'INSERT INTO {full_table_name} ({columns})\nVALUES {values}'
+                    with self.printer.print_msg(query):
```

Comment:
@wangxiaoyou1993 @Luishfs why are we printing here and not checking the value of `verbose`?

Reply:
will fix